src/ralph/reports/tests.py

```diff
@@ -177,6 +178,13 @@ def test_asset_relation(self):
         ]
         self.assertEqual(report_result, result)
 
+    def test_num_queries(self):
```

Comment:
btw in this kind of test it'd be cool to provide sufficient amount of test data (ex. 1000 licences)